airflow/ui/src/router.tsx

```diff
@@ -51,6 +52,10 @@ export const router = createBrowserRouter(
           element: <Events />,
           path: "events",
         },
+        {
+          element: <XCom />,
+          path: "xcoms",
```

Comment:
```suggestion
          path: "xcom",
```

Reply:
Other names at the top level path are plural like variables, events, dags etc. Does this need to be xcom without plural?

Reply:
Yeah we can leave this.